CHANGES.md

```diff
@@ -5,6 +5,7 @@
 ##### Additions :tada:
 
 - Add support for the CSS `line-height` specifier in the `font` property of a `Label`. [#8954](https://github.com/CesiumGS/cesium/pull/8954)
+- `Viewer` now has default pick handling for `Cesium3DTileFeature` data and will display its properties in the default Viewer `InfoBox` as well as set `Viewer.selectedEntity` to a transient Entity instance representing the data.
```

Comment:
Don't forget a link to the PR in changes.md.

Reply:
Chicken and egg, thanks.